components/esp_modem/test/target_ota/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# The following lines of boilerplate have to be in your project's CMakeLists
+# in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.8)
+
+set(EXTRA_COMPONENT_DIRS "../.." "../../examples/modem_tcp_client/components")
```

Comment:
Can idf_component.yml be used instead of instead of EXTRA_COMPONENT_DIRS in cmakelist?

Reply:
I will check if the component manager support this and convert it